CMakeLists.txt

```diff
@@ -67,13 +67,19 @@ ENDIF()
 find_package(OpenGL REQUIRED)
 find_package(OpenAL REQUIRED)
 find_package(Bullet REQUIRED)
-find_package(SFML 2 COMPONENTS system window graphics network REQUIRED)
 find_package(MAD REQUIRED)
 find_package(GLM REQUIRED)
 find_package(LibSndFile REQUIRED)
+find_package(SDL2 REQUIRED)
```

Comment:
Just to note: Ubuntu 16.04 includes sdl2-config.cmake but earlier releases don't, they'll need to build SDL from source.